homeassistant/components/hardware/models.py

```diff
@@ -0,0 +1,33 @@
+"""Models for Hardware."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Protocol
+
+from homeassistant.core import HomeAssistant, callback
+
+
+@dataclass
+class BoardInfo:
+    """Board info type."""
+
+    manufacturer: str
+    model: str | None
+    revision: str | None
```

Comment:
We should add the board name that was retrieved from Hassio

Reply:
Added in bf607df